beesdoo_base/wizard/__init__.py

```diff
@@ -1,3 +1 @@
-from . import member_card
-from . import partner
 from . import portal_wizard
```

Comment:
These are ancestral changes https://github.com/beescoop/Obeesdoo/commit/7c6a7ff4ac3191813b3aaa222949a4c8aa430024, I think they could be removed after a little investigation.

Reply:
changes removed